integration_tests/suite/test_http_callback.py

```diff
@@ -917,39 +909,64 @@ def test_given_user_tenant_dont_trigger_on_other_tenant_event(self, subscription
             },
         )
 
-        assert_that(
-            calling(until.assert_).with_args(
-                self.make_third_party_verify_callback(
-                    request={'method': 'GET', 'path': '/test'}, count=1, exact=True
-                ),
-                timeout=10,
-                interval=0.5,
+        # trigger control webhook
+        self.bus.publish(
+            trigger_event(),
+            headers={
+                'name': TRIGGER_EVENT_NAME,
+                'tenant_uuid': USERS_TENANT,
+            },
+        )
+
+        until.assert_(
+            self.make_third_party_verify_callback(
+                request={'method': 'GET', 'path': '/test'}, count=1, exact=True
```

Comment:
If the other tenant had triggered, would the test falsely pass? I guess we need some way to distinguish the first and second request.

Reply:
Good point, yes it does, I have reworked the tests